guides/source/active_record_migrations.md

```diff
@@ -344,8 +345,7 @@ create_table :products do |t|
 end
 ```
 
-which creates a `products` table with a column called `name` (and as discussed
-below, an implicit `id` column).
+which creates a `products` table with a column called `name`.
```

Comment:
Should we remove the "(and as discussed below, an implicit `id` column)" part?

Reply:
Yes. It is redundant. The very next sentence states just that.